src/components/NavBar.tsx

```diff
@@ -75,6 +75,9 @@ const NavBar: React.FC = () => {
     setAnchorEl(null);
   };
 
+  const eventsDocsLink =
```

Comment:
Wait, I don't know how I feel about this part, it seems weird for a nav link to go external. I think it's worth our effort to add a calendar to the site tbh

Reply:
Good point, my concern was having 2 sources of information for the events. I was thinking that it would be really easy to desync the actual event schedule and whatever we have on the website. It would be best if we could somehow read the google doc and then display that on the page. Im sure theres some npm module for that or worst case we can simply scrape it.